actions.yml

```diff
@@ -130,3 +130,14 @@ github/codeql-action/analyze:
     contents: read
     pull-requests: read
     security-events: write
+ridedott/merge-me-action:
```

Comment:
https://github.com/ridedott/merge-me-action/blob/7337d121e5e14af8ff1485eaab95caeeea2cf24d/src/common/getPullRequestInformation.ts#L95-L121
Probably, need to  `pull-requests: read` addtionarly.